__tests__/app.test.js

```diff
@@ -1,12 +1,14 @@
-import app from '../src/index';
+/* eslint-disable import/extensions */
+import app from '../src/App.js';
```

Comment:
Welldone @Yasholma , but i am getting error while running the tests, were you able to run it successfully, if so how did you go about doing it?

Reply:
The test won't pass yet because I used a dummy data will rewrite a new test when we start work properly.

Reply:
Alright @Yasholma